### PR TITLE
Fix `application/x-www-form-urlencoded` content-type header check

### PR DESCRIPTION
**Pull request by cantoniazzi**
**Fixes** # (*[issue](https://github.com/rudderlabs/rudder-server/issues/1463)*)

## Type of change
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
## Checklist:
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added unit tests for the code
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

**Comment by chandumlg** [comment]
@cantoniazzi Thank you very much for the PR.

Can you please sign the contributor licence agreement. https://github.com/rudderlabs/rudder-server/blob/master/CONTRIBUTING.md

**Comment by cantoniazzi** [comment]
> @cantoniazzi Thank you very much for the PR.
> 
> Can you please sign the contributor licence agreement. https://github.com/rudderlabs/rudder-server/blob/master/CONTRIBUTING.md

Sure. Done @chandumlg 🚀

File: gateway/webhook/webhook.go [hunk 1]
```diff
@@ -138,7 +138,7 @@ func (webhook *HandleT) RequestHandler(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 	contentType := r.Header.Get("Content-Type")
-	if strings.ToLower(contentType) == "application/x-www-form-urlencoded" {
+	if strings.Contains(strings.ToLower(contentType), "application/x-www-form-urlencoded") {
```
Comment (lvrach) [inline gateway/webhook/webhook.go:141]: [non-blocking]
Ideally, if we want to be exact, we should use [mine.ParseMediaType](https://pkg.go.dev/mime#ParseMediaType) to parse `Content-Type` [contents](https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types#structure_of_a_mime_type)

**Review by lvrach — APPROVED** [review]

**Comment by lvrach** [comment]
@cantoniazzi thank you for the PR  🙇

**Comment by cantoniazzi** [comment]
@lvrach @chandumlg, I couldn't simulate my improvement in my local environment, but given the problem I had and analyzing this specific piece of code, I believe this will solve it. Do you agree with this?

**Comment by cantoniazzi** [comment]
> @lvrach @chandumlg @psrikanth88, I couldn't simulate my improvement in my local environment, but given the problem I had and analyzing this specific piece of code, I believe this will solve it. Do you agree with this?

Guys, we were able to test the change correctly using the image generated in pr in our cloud and it worked. 👯 

image (2)

**Pull request merged**